enrollment-server/src/main/java/com/wultra/app/docverify/zenid/provider/ZenidDocumentVerificationProvider.java

```diff
@@ -499,4 +544,29 @@ private DocumentVerificationStatus toStatus(ZenidWebInvestigateResponse.StateEnu
         }
     }
 
+    @Nullable
+    private DocumentType toDocumentType(ZenidSharedMineAllResult.DocumentRoleEnum documentRoleEnum) {
+        switch (documentRoleEnum) {
+            case DRV:
+                return DocumentType.DRIVING_LICENSE;
+            case IDC:
+                return DocumentType.ID_CARD;
+            case PAS:
+                return DocumentType.PASSPORT;
+            default:
+                return null;
```

Comment:
I am not sure about the `null` value. I would like to suggest to add a new `DocumentType.UNKNOWN` enum value, what do you think?

Reply:
Fixed